### ISSUE TITLE
Document which directories contain precious data

### ISSUE THREAD

**Issue by jernst** [solid/node-solid-server#821]
Which of the directories contains data that needs to be backed up, and which can be discarded because it can be easily recreated? And once identified, how to I backup and restore correctly?

**Comment by kjetilk** [comment]
Yeah, this would be a good addition to the FAQ. @Mitzi-Laszlo can you follow up?

**Comment by Mitzi-Laszlo** [comment]
There is more on this conversation here https://forum.solidproject.org/t/which-of-the-directories-contains-data-that-needs-to-be-backed-up-and-which-can-be-discarded-because-it-can-be-easily-recreated-and-once-identified-how-to-i-backup-and-restore-correctly/533

**Comment by jernst** [comment]
Same comment as the other bug: why was this closed? Fixed? Wontfix? 

Presumably an issue tracker is better at tracking issues than a discussion board?

**Comment by kjetilk** [comment]
So, you didn't get much feedback from us, we've been pretty swamped, sorry about that. However, since it is something that others could figure out, and it hasn't been picked up by other community members here, we figured it might get more response in the forum. So, mostly a pragmatic thing, hoping there would be more feedback, and once figured out, it is more a likely addition to the FAQ than the module documentation.

**Comment by csarven** [comment]
@kjetilk I beg to differ. Announcing at different channels to raise awareness is completely orthogonal. That's not at all grounds to close the issue IMO.

**Comment by NoelDeMartin** [comment]
Is there any updates on this? I'm configuring backups and I'm not sure what I need to save.

So far, looking at `.gitignore`, I'm backing up `config`, `data` and `.db` folders. There's also an `accounts` folder ignored, but it doesn't exist in my server so I guess it's deprecated or doesn't apply to me (I have multiuser disabled).

As per how to restore, I suppose just copying over those files should be enough. And of course, any ssl keys should also be kept safe (but these don't need to be included in backups because they won't change).

**Comment by kjetilk** [comment]
AFAIK, the ones that you've found are the important ones. Some of it, e.g. `.db`, are NSS-specific things. The more general topic of data portability is something that falls under the [data interop panel](https://github.com/solid/data-interoperability-panel), so you might want to stroll over there :-)
